lib/miq_automation_engine/engine/miq_ae_engine/miq_ae_object_lookup.rb

```diff
@@ -0,0 +1,60 @@
+module MiqAeEngine
+  module MiqAeObjectLookup
+    def initialize_obj_entries
+      @lookup_table = []
+    end
+
+    def add_obj_entry(fqns, klass, instance, object)
+      parts = fqns.split('/')
+      domain = parts.shift
+      namespace = parts.join('/')
+
+      @lookup_table << {:domain    => domain.downcase,
```

Comment:
@gmcculloug 
The automate model is case insensitive so we always lowercase the names when adding to the table.
But when we do the glob match we have to do a case-insensitive search
File.fnmatch(instance, entry[:instance], File::FNM_CASEFOLD)
Since the subst string is enter by the user they could use mixed case but we convert everything to lower case when doing a match. The fnmatch function needs the FNM_CASEFOLD to make it case insensitive.
If there are multiple instances we use the first one. If there is no match we return a nil and the subst call fails